### PR TITLE
Stylistic revisions of man page

### PR DESCRIPTION
**Pull request by jordydickinson**

### CODE REVIEW

File: man/man8/pam_panic.8 [hunk 1]
```diff
@@ -124,43 +117,36 @@ Access was granted\&.
...
 .PP
 /lib/*/security/pam_panic\&.so
 .RS 4
-Shared object file which does all the work\&.
-.RE
-.PP
-/usr/share/man/*/man8/pam_panic\&.8\&.gz
-.RS 4
-The man pages\&.
+This kernel module\&.
```
Comment (Bandie) [inline man/man8/pam_panic.8]: This is not a kernel module.

File: man/man8/pam_panic.8 [hunk 2]
```diff
@@ -32,79 +32,72 @@ pam_panic \- PAM module for access control using removable media. With panic fun
...
 
 
-.SH "HOW DO I USE IT"
+.SH "USAGE"
 .PP
-To activate the module you have to change some file(s) in \fI/etc/pam.d/\fR\&.
-There you'll see different services, programs and other config files which use the PAM modules\&.
-Speaking of the general authentication configs, if you use Ubuntu "\fIcommon-auth\fR" might be interesting for you\&.
-For other distributions you might want to look into the distribution's wiki\&.
+To activate the module you have to configure PAM\&. See \fBpam\&.conf(5)\fR for details\&.
```
Comment (jordydickinson) [inline man/man8/pam_panic.8]: A lot of the content here was dropped. Nothing wrong with it per se, but I wasn't sure what it was saying and thus couldn't incorporate it into my revisions. This should probably be addressed if the information was important.

Comment (Bandie) [inline man/man8/pam_panic.8]: Good idea!

File: man/man8/pam_panic.8 [hunk 3]
```diff
@@ -124,43 +117,36 @@ Access was granted\&.
...
-.RS 4
-The man pages\&.
+This kernel module\&.
 .RE
 
 
 .SH "BUGS"
 .PP
-Please report bugs or do merge requests at https://github.com/Bandie/pam_panic 
+Please report bugs and send pull requests to <https://github\&.com/Bandie/pam_panic>\&.
```
Comment (Bandie) [inline man/man8/pam_panic.8]: Thanks!

File: man/man8/pam_panic.8 [hunk 4]
```diff
@@ -32,79 +32,72 @@ pam_panic \- PAM module for access control using removable media. With panic fun
...
 .RS 4
-This device UUID is used for authentication (auth key)\&.
+The UUID of the device to be used for authentication (the auth key)\&.
 .PD 0
 .PP
 .PD 1
-It is required to have a GPT formated device with at least one partition on it\&.
-The UUID of a GPT formated device looks like "12345678-9ABC-DEF0-1234-56789ABCDEF0"\&. 
+The device must have a GPT formated device with at least one partition on it\&.
+The UUID of a GPT-formated device looks like "12345678-9ABC-DEF0-1234-56789ABCDEF0"\&.
```
Comment (jordydickinson) [inline man/man8/pam_panic.8]: I missed this, but all occurrences of "GPT formated" should be "GPT-formatted". The hyphen isn't critical, but I like it personally.

File: man/man8/pam_panic.8 (hunk 1 above)
Comment (jordydickinson) [reply man/man8/pam_panic.8]: Probably "PAM module" then

File: man/man8/pam_panic.8 (hunk 4 above)
Comment (Bandie) [reply man/man8/pam_panic.8]: L47 should have it too then <.<

File: man/man8/pam_panic.8 (hunk 2 above)
Comment (Bandie) [reply man/man8/pam_panic.8]: I agree.

**Pull request merged**